python/bottle/containerWithTests/Tests/functional_tests/test_functional.py

```diff
@@ -16,15 +16,17 @@ def setUp(self):
 
 	def test_selenium(self):
 		webAppUrl = pytest.config.getoption('webAppUrl')
-		num_retries = 5
-		for i in range(num_retries):
+		start_timestamp = time.time()
+		end_timestamp = start_timestamp + 60*10		
+		while True:
 			try:
 				response = self.driver.get(webAppUrl)
 				title = self.driver.title
 				self.assertIn("Home Page - Python Bottle Application", title)
 				break
 			except AssertionError:
-				if(i == (num_retries - 1)):
+                current_timestamp = time.time()
```

Comment:
alignment (spacing) is bad

Reply:
It is a diff problem, alignment looks fine in vscode

Reply:
@issacnitin you are using tabs for indentation and hence this problem. Use whitespace only

Reply:
All python files have tabs for indentation, so I'm leaving it as is.